internal/runbits/requirements/requirements.go

```diff
@@ -355,6 +423,85 @@ func (r *RequirementOperation) updateCommitID(commitID strfmt.UUID) error {
 	return nil
 }
 
+func (r *RequirementOperation) shouldPromptForSecurity(vulnerabilities *model.VulnerabilityIngredient) bool {
+	if !r.Config.IsSet(constants.SecurityPromptConfig) {
+		if err := r.Config.Set(constants.SecurityPromptConfig, promptDefault); err != nil {
+			multilog.Error("Failed to set security prompt config: %v", err)
+		}
+	}
+
+	if !r.Config.IsSet(constants.SecurityPromptLevelConfig) {
+		if err := r.Config.Set(constants.SecurityPromptLevelConfig, promptDefaultLevel); err != nil {
+			multilog.Error("Failed to set security prompt level config: %v", err)
+		}
+	}
+
+	prompt := r.Config.GetBool(constants.SecurityPromptConfig)
+	promptLevel := r.Config.GetString(constants.SecurityPromptLevelConfig)
+
+	switch promptLevel {
+	case vulnModel.SeverityCritical:
+		return prompt && len(vulnerabilities.Vulnerabilities.Critical) > 0
+	case vulnModel.SeverityHigh:
+		return prompt &&
+			(len(vulnerabilities.Vulnerabilities.Critical) > 0 ||
+				len(vulnerabilities.Vulnerabilities.High) > 0)
+	case vulnModel.SeverityMedium:
+		return prompt &&
+			(len(vulnerabilities.Vulnerabilities.Critical) > 0 ||
+				len(vulnerabilities.Vulnerabilities.High) > 0 ||
+				len(vulnerabilities.Vulnerabilities.Medium) > 0)
+	case vulnModel.SeverityLow:
+		return prompt &&
+			(len(vulnerabilities.Vulnerabilities.Critical) > 0 ||
+				len(vulnerabilities.Vulnerabilities.High) > 0 ||
+				len(vulnerabilities.Vulnerabilities.Medium) > 0 ||
+				len(vulnerabilities.Vulnerabilities.Low) > 0)
+	}
+
+	return r.Config.GetBool(constants.SecurityPromptConfig)
+}
+
+func (r *RequirementOperation) promptForSecurity(out output.Outputer, vulnerabilities *model.VulnerabilityIngredient) (bool, error) {
+	out.Print(locale.Tr("warning_vulnerable_simple", strconv.Itoa(vulnerabilities.Vulnerabilities.Length())))
```

Comment:
Again, please don't use `out.Print()` with strings. I think you can safely replace all of your calls with `out.Notice()` for this ticket.

Reply:
In testing this, `Notice` does not produce the desired output when passing it a slice. Should I use the `output.Prepare` function here? I'm unfamiliar with how that works.

Reply:
Whoops, sorry I was looking at the wrong line. When I pass a slice on line 494 I'm using `Print` but `Notice` is fine here.